src/lib/gqlEasAttestation/index.ts

```diff
@@ -1,6 +1,9 @@
 import { optimism, optimismSepolia } from "wagmi/chains";
 
 export const API_URL_MAPPING: { [key: string]: string } = {
-  [optimismSepolia.id]: "https://optimism-sepolia.easscan.org/graphql",
-  [optimism.id]: "https://optimism.easscan.org/graphql",
+	[optimismSepolia.id]: "https://optimism-sepolia.easscan.org/graphql",
+	[optimism.id]: "https://optimism.easscan.org/graphql",
 } as const;
+
+export const EVENT_SCHEMA_ID =
```

Comment:
No need to export this EVENT_SCHEMA_ID here, you can use the ID created in components/createEventAttestationButton, because this id will change if we change event schema.

Reply:
yep